Gruntfile.js

```diff
@@ -188,6 +177,21 @@ module.exports = (grunt) => {
     }
   });
 
+  grunt.registerTask('checkPrettier', 'Verify that all files are correctly formatted.', function() {
```

Comment:
Nit: Current task naming convention seems to be `kebab-case`.

```suggestion
  grunt.registerTask('check-prettier', 'Verify that all files are correctly formatted.', function() {
```

Reply:
We currently already have different naming conventions. e.g.
`clickParallel`
`travisnpmpublish`
`browserify-common`
So I don't mind because we might change that later.